lib/serde-fallback.js

```
@@ -2,691 +2,770 @@
+  //   character - a character to check
+  //
+  isDigit(character) {
+    return (
+      this.isInitialDigit(character) || character === 'e' || character === 'E'
+    );
+  }
+
+  // Check if a given character is a binary digit
+  //   character - a character to check
+  //
+  isBinaryDigit(character) {
+    return character === '0' || character === '1';
+  }
+
+  // Check if a given character is an octal digit
+  //   character - a character to check
+  //
+  isOctalDigit(character) {
+    return character >= '0' && character <= '7';
+  }
+
+  // Check if a given character is a decimal digit
+  //   character - a character to check
+  //
+  isDecimalDigit(character) {
+    return character >= '0' && character <= '9';
+  }
+
+  // Check if a given character is a hexadecimal digit
+  //   character - a character to check
+  //
+  isHexadecimalDigit(character) {
+    return (
+      (character >= '0' && character <= '9') ||
+      (character >= 'a' && character <= 'f') ||
+      (character >= 'A' && character <= 'F')
+    );
+  }
+
+  // Check if a given character is a quote character
+  //   character - a character to check
+  //
+  isQuoteCharacter(character) {
+    return character === "'" || character === '"';
+  }
+
+  // Check if a given character can be the first character of an identifier
+  //   character - a character to check
+  //
+  isInitialIdentifierCharacter(character) {
+    return (
+      (character >= 'a' && character <= 'z') ||
+      (character >= 'A' && character <= 'Z') ||
+      (character === '_' || character === '$')
+    );
+  }
+
+  // Check if a given character can be a part of an identifier
+  //   character - a character to check
+  //
+  isIdentifierCharacter(character) {
+    return (
+      this.isInitialIdentifierCharacter(character) ||
+      this.isDecimalDigit(character)
+    );
+  }
+
+  // Skip whitespace and comments
+  //
+  skipClutter() {
+    this.skipWhitespaceNewline();
+    while (this.lookahead() === '/') {
```

This loop doesn't terminate when there is a single slash sign, for example:
```js
'str' /
```
It should throw an error in such cases, as it did before.